static/src/javascripts/projects/common/modules/video/youtube-tracking.js

```diff
@@ -0,0 +1,81 @@
+define([
+    'common/utils/mediator',
+    'lodash/collections/forEach',
+    'common/utils/config',
+    'common/modules/video/ga-helper'
+], function (
+    mediator,
+    forEach,
+    config,
+    gaHelper
+) {
+
+    function eventAction() {
+        return 'video content';
+    }
+
+    function buildEventId(event, videoId) {
+        return event + ':' + videoId;
+    }
+
+    function initYoutubeEvents(videoId) {
+
+        var ga = window.ga,
+            gaTracker = config.googleAnalytics.trackers.editorial;
```

Comment:
This is a tiny nit, but can you have a declaration for each `var` rather that comma separate them.